features-json/css-has.json

```diff
@@ -0,0 +1,256 @@
+{
+  "title":":has() CSS relational pseudo-class",
+  "description":"Only select elements containing specified content. for example, `a:has(>img)` select all `<a>` elements that contains a `<img>` child",
+  "spec":"https://drafts.csswg.org/selectors-4/#relational",
+  "status":"wd",
```

Comment:
`:has()` is only in the ED, not [the WD](https://www.w3.org/TR/selectors4/), so this should be `"unoff"`